src/helpers/calculateTimeLeft.ts

```diff
@@ -9,8 +9,8 @@ const calculateTimeLeft = (date: string): TimeLeft => {
   const difference =
     Number(new Date(date).getTime()) - Number(new Date().getTime());
 
-  const daysLeft = Math.floor(difference / (1000 * 60 * 60 * 24) + 1);
-  const hoursLeft = Math.floor((difference / (1000 * 60 * 60)) % 24) - 6;
+  const daysLeft = Math.floor(difference / (1000 * 60 * 60 * 24));
+  const hoursLeft = Math.floor((difference / (1000 * 60 * 60)) % 24) + 18;
```

Comment:
can you add a comment in the code on the line above saying why you've added the 18

Reply:
To equate for the 18.00 hours but now realised this was incorrect - the first solution works.